recipe/meta.yaml

```
@@ -1,33 +1,35 @@
-{% set version = "0.9.8" %}
+{% set version = "1.2.0" %}
 
 package:
   name: requests-cache
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/r/requests-cache/requests_cache-{{ version }}.tar.gz
-  sha256: eaed4eb5fd5c392ba5e7cfa000d4ab96b1d32c1a1620f37aa558c43741ac362b
+  sha256: db1c709ca343cc1cd5b6c8b1a5387298eceed02306a6040760db538c885e3838
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<38]
 
 requirements:
   host:
-    - python >=3.7
-    - poetry
+    - python
+    - poetry-core
     - pip
+    - setuptools
+    - wheel
 
   run:
-    - python >=3.7
-    - appdirs >=1.4.4
+    - python
     - attrs >=21.2
     - cattrs >=22.2
     - itsdangerous >=2.0
+    - platformdirs >=2.5
     - pyyaml >=5.4
```

```suggestion
    - pyyaml >=6.0.1
```
https://github.com/requests-cache/requests-cache/blob/v1.2.0/pyproject.toml#L64C57-L64C62